packages/compiler/core/types.ts

```diff
@@ -534,7 +534,7 @@ export interface TypeInstantiationMap {
  * AST types
  */
 export enum SyntaxKind {
-  CadlScript,
+  TypeSpecScript,
```

Comment:
Ok not doing back-compat for enum members.

Reply:
Why don't we want to export back-compat for enum while doing for others? @bterlson